lib/crypt_keeper/provider/active_support.rb

```diff
@@ -0,0 +1,51 @@
+require "active_support/message_encryptor"
+
+module CryptKeeper
+  module Provider
+    class ActiveSupport < Base
+      attr_reader :encryptor
+
+      # Public: Initializes the encryptor
+      #
+      # options - A hash, :key and :salt are required
+      #
+      # Returns nothing.
+      def initialize(options = {})
+        key  = options.fetch(:key)
+        salt = options.fetch(:salt)
+
+        @encryptor = ::ActiveSupport::MessageEncryptor.new \
+          ::ActiveSupport::KeyGenerator.new(key).generate_key(salt)
```

Comment:
I need to check into the usage/security more, but we can also use the Rails secret here, so something like:

```ruby
secret = options.fetch :secret do
  key  = options.fetch(:key)
  salt  = options.fetch(:salt)
  ::ActiveSupport::KeyGenerator.new(key).generate_key(salt)
end

@encryptor = ::ActiveSupport::MessageEncryptor.new(secret)
```

Which would allow simpler model setup:

```ruby
class User < ApplicationRecord
  crypt_keeper :ssn, encryptor: :active_support, secret: Rails.application.secrets.secret_key_base
end
```